src/scancode/cli.py

```diff
@@ -223,20 +232,39 @@ def reindex_licenses(ctx, param, value):
 number of files processed. Use --verbose to display file-by-file progress.
 '''
 
+formats = ('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')
```

Comment:
May be we have a disconnect here: to me each of `('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')` should become a format plugin... e.g. there should not be both formats from plugins and formats that are built in.